usaspending_api/broker/management/commands/load_fpds_from_broker.py

```diff
@@ -0,0 +1,82 @@
+from os import environ
+from django.core.management.base import BaseCommand
+import logging
+import re
+import psycopg2
+
+from usaspending_api.data_load.fpds_loader import run_fpds_load
+from usaspending_api.common.retrieve_file_from_uri import RetrieveFileFromUri
+
+logger = logging.getLogger("console")
+
+BROKER_CONNECTION_STRING = environ["DATA_BROKER_DATABASE_URL"]
+
+CHUNK_SIZE = 50000  # Completely arbitrary and not backed by any testing, this can likely go higher
+
+
+class Command(BaseCommand):
+    help = "Sync USAspending DB FPDS data using Broker for new or modified records and S3 for deleted IDs"
+
+    @staticmethod
+    def get_fpds_transaction_ids_from_date(date):
+        with psycopg2.connect(dsn=BROKER_CONNECTION_STRING) as connection:
+            db_cursor = connection.cursor()
+            db_query = "SELECT detached_award_procurement_id FROM detached_award_procurement WHERE updated_at >= %s;"
+            db_args = [date]
+
+            db_cursor.execute(db_query, db_args)
+            db_rows = [id[0] for id in db_cursor.fetchall()]
+
+        return db_rows
+
+    def load_fpds_from_date(self, date):
+        if not date:
+            return
+        # Not doing any batching here, since if the array of ids is blowing you up I don't know how to chunk that on
```

Comment:
The loading stage batches itself. As for batching the ids from the date query, do you know a way to do that?

Reply:
I just ran a --reload-all, and it was able to hold on to 52208277 without batching

Reply:
locally?
how much memory does it take to run the script while holding those ids?

Reply:
locally, and my bad, it was taking gigabytes of RAM. This will need to be fixed, but frankly I have no idea how.

Reply:
Might need to be creative, there should be a streaming API for sending DB results back as a stream instead of loading all of the results. It would maintain an open connection but it is a clean way when run in a generator.

Another is to add order and limit to the query, but it will probably add time.